modules/gapi/src/streaming/onevpl/utils.cpp

```diff
@@ -429,4 +432,69 @@ std::string ext_mem_frame_type_to_cstr(int type) {
 } // namespace wip
 } // namespace gapi
 } // namespace cv
+
+namespace {
+namespace cfg {
+// FIXME: Move OneVPL arguments parser to a single place
+typename cv::gapi::wip::onevpl::CfgParam create_from_string(const std::string &line) {
```

Comment:
I don't like this function either.
it doesn't relates to VPL CORE utility, because it assumes to interact with user input

But if you want this function- - please
1) move it into cfg_params.hpp as static method of CfgParams
2) get hardcoded separator ':' and upgrade it as function argument with default value = ':'
3) and rename it on something like "parse_from_pair"

Reply:
Going to rework this since we need some pointers for inference